conf/rocks-inode-bloom.ini.template

```diff
@@ -0,0 +1,55 @@
+# This is a RocksDB option file.
```

Comment:
This is a template using optimizations such as Bloom filters as described in the readme Metastore.md